packages/design-tokens/src/scss/generated/_typography.scss

```diff
@@ -151,72 +151,72 @@ $body-xsmall-text-regular: (
 
 $display-large-text: (
     mobile: (
-        font-family: "'Nunito Sans', sans-serif",
+        font-family: "'Ebony', sans-serif",
         font-size: 2rem,
         font-style: normal,
-        font-weight: 800,
+        font-weight: 600,
```

Comment:
This font family has variable font weights, which doesn't correspond with [the standard naming](https://github.com/lmc-eu/spirit-design-system/blob/main/exporters/scss/src/js/normalizers/weight.ts).
Ebony Semibold has weight `400`. 🙈 
We'll solved this case in the SCSS exporter, meanwhile this PR can be merged, because Jobs doesn't use any other variant of this font.